smt/utils/misc.py

```diff
@@ -108,6 +108,35 @@ def compute_rms_error(sm, xe=None, ye=None, kx=None):
         return num / den
 
 
+def compute_pva_error(sm, xe, ye):
+    ye = ye.reshape((xe.shape[0], 1))
+    N = len(ye)
+    ye2 = sm.predict_values(xe)
+    variance = sm.predict_variances(xe)
+
+    error = (ye2 - ye) ** 2 / variance
+    pva = np.sum(error) / N
+    return pva
+
+
+def compute_rmse_error(sm, xe, ye):
```

Comment:
This function name reads like "compute root mean squared error error". I suggest to remove "_error" from the functions names: `compute_pva`, `compute_rmse` and `compute_q2`